test/views/CompletedReturnsViewSpec.scala

```diff
@@ -175,7 +175,7 @@ class CompletedReturnsViewSpec extends ViewBaseSpec {
     }
   }
 
-  "Rendering the VAT Returns page with only one returned year" should {
+  "Rendering the VAT Returns page with only one returned year" when {
```

Comment:
Should be 'return year' to match model and make it easier to understand. Maybe above the two scenarios should also be nested and say something like 'Rendering the VAT Returns page with multiple return years' to match this?